web_pivot_computed_measure/static/src/pivot/pivot_model.esm.js

```diff
@@ -275,17 +279,20 @@ patch(PivotModel.prototype, {
         const fieldNames = Object.keys(this.metaData.fields);
         for (const fieldName of fieldNames) {
             const field = this.metaData.fields[fieldName];
-            if (field.__computed_id) {
-                const cm = this._computed_measures.find((item) => {
-                    return item.id === field.__computed_id;
-                });
+            if (field?.__computed_id) {
+                const cm = this._computed_measures.find(
+                    (item) => item.id === field.__computed_id
+                );
                 if (!cm) {
                     delete this.metaData.fields[fieldName];
                     delete this.metaData.measures[fieldName];
                     this.metaData.activeMeasures = this.metaData.activeMeasures.filter(
                         (item) => item !== fieldName
                     );
                 }
+            } else if (this.env.debug === "1") {
+                // You can console log for debugging metaData.fields
+                // or fieldName or __computed_id
```

Comment:
```suggestion
            }
```
Remove unnecesary code